tests/foreman/ui_airgun/test_hostcollection.py

```diff
@@ -635,3 +702,89 @@ def test_negative_hosts_limit(session, module_org, module_loc):
+
+    :id: e5d882e0-3520-4cb6-8629-ef4c18692868
+
+    :Steps:
+        1. Running dnf upload profile to sync module streams from hosts to Satellite
+        2. Navigating to host_collection
+        3. Installing the module stream duck
+        4. Verifying that remote job get passed
+        5. Verifying that package get installed
+
+    :expectedresults: Module-Stream was successfully installed on all the hosts
+        in host collection
+
+    :CaseLevel: System
+    """
+    with session:
+        _run_remote_command_on_content_hosts(
+            'dnf -y upload-profile',
+            vm_content_hosts_module_stream
+        )
+        session.organization.select(org_name=module_org.name)
+        result = session.hostcollection.manage_module_streams(
+            vm_host_collection_module_stream.name,
+            action_type="Install",
+            module_name=FAKE_3_CUSTOM_PACKAGE_NAME,
+            stream_version="0"
+        )
+
+        assert result['overview']['job_status'] == 'Success'
+        assert result['overview']['job_status_progress'] == '100%'
+        assert int(result['overview']['total_hosts']) == 2
+        assert _is_package_installed(vm_content_hosts_module_stream, FAKE_3_CUSTOM_PACKAGE)
+
+
+@tier3
+@upgrade
+def test_positive_install_modular_errata(
+        session, module_org, vm_content_hosts_module_stream, vm_host_collection_module_stream):
+    """Install Modular Errata generated from module streams.
+
+    :id: 8d6fb447-af86-4084-a147-7910f0cecdef
+
+    :Steps:
+        1. Generated errata by installing and downgrading the Module Stream Kangaroo
+        2. Running dnf upload-profile
+        3. Installed the modular errata by 'remote execution'
+        4. Verifying that latest package get installed
+
+    :expectedresults: Modular Errata should get installed on all hosts in host
+        collection.
+
+    :CaseLevel: System
+    """
+    with session:
+        stream = "0"
+        version = "20180704111719"
+        _module_install_command = 'dnf -y module install {}:{}:{}'.format(
+            FAKE_4_CUSTOM_PACKAGE_NAME,
+            stream,
+            version
+        )
+        _run_remote_command_on_content_hosts(
+            _module_install_command,
+            vm_content_hosts_module_stream
+        )
+        _run_remote_command_on_content_hosts(
+            'dnf -y upload-profile',
+            vm_content_hosts_module_stream
+        )
+        session.organization.select(org_name=module_org.name)
+        result = session.hostcollection.install_errata(
+            vm_host_collection_module_stream.name,
+            FAKE_0_MODULAR_ERRATA_ID,
+            install_via='via remote execution'
+        )
+        assert result['job_status'] == 'Success'
+        assert result['job_status_progress'] == '100%'
+        assert int(result['overview']['total_hosts']) == 2
+        assert _is_package_installed(vm_content_hosts_module_stream, FAKE_4_CUSTOM_PACKAGE)
```

Comment:
Is it possible to check if correct version of package is installed?
I think this assert will pass right after `dnf -y module install`, even if errata couldn't be installed.

Reply:
@mirzal when we do dnf -y module install it installs the specific version kangaroo-0.2-1. noarch module stream which eventually generates the errata. 

Now, when we check this using _is_package_installed, I'm passing version as kangaroo-0.3-1.noarch
which make sure that it installed the errata. Please correct me if I'm understood your question incorrectly

Reply:
I thought you are checking `FAKE_4_CUSTOM_PACKAGE_NAME`, which doesn't have version. In that case, all is good. Thanks.